Sources/_MatchingEngine/Utility/TypeConstruction.swift

```diff
@@ -128,13 +128,13 @@ public enum TypeConstruction {
         // Open existential on each element type.
         func initializeElement<T>(_ element: T) {
           currentElementAddressUnaligned =
-          currentElementAddressUnaligned.roundedUp(toAlignmentOf: T.self)
+            currentElementAddressUnaligned.roundedUp(toAlignmentOf: T.self)
```

Comment:
```suggestion
          currentElementAddressUnaligned.roundedUp(toAlignmentOf: T.self)
```

Reply:
Why is that? This and the previous line are part of the same statement. So this is line wrapping according to Google Swift Style Guide. Is there a different style guide that wants this done differently?

Reply:
Oh, I didn't see that. Argh, I think Xcode refuses to indent it